extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -301,4 +302,59 @@ export class RemoteQueriesManager extends DisposableObject {
       queryItem.status = QueryStatus.Failed;
     }
   }
+
+  public async exportVariantAnalysisResults(): Promise<void> {
+    const queryId = this.interfaceManager.getCurrentQueryId();
+    const queryHistoryItem = this.qhm.getCurrentQueryHistoryItem();
+
+    if (!queryId || !queryHistoryItem || !queryHistoryItem.completed || queryHistoryItem.t !== 'remote') {
+      throw new Error('No variant analysis results currently open. To open results, click an item in the query history view.');
+    }
+
+    void this.logger.log(`Exporting variant analysis results for query: ${queryId}`);
+    const query = queryHistoryItem.remoteQuery;
+    const analysesResults = this.analysesResultsManager.getAnalysesResults(queryId);
+
+    const credentials = await Credentials.initialize(this.ctx);
```

Comment:
Shall we move this closer to its usage? We don't need to get credentials if we're going to just create local files.